suite-native/device/src/hooks/useDeviceConnect.tsx

```diff
@@ -0,0 +1,93 @@
+import { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+
+import { useNavigation } from '@react-navigation/native';
+
+import {
+    ConnectDeviceStackParamList,
+    ConnectDeviceStackRoutes,
+    RootStackParamList,
+    RootStackRoutes,
+    StackToStackCompositeNavigationProps,
+} from '@suite-native/navigation';
+import {
+    acquireDevice,
+    selectDeviceRequestedPin,
+    selectIsDeviceConnectedAndAuthorized,
+    selectIsDeviceConnectedButNotInitialized,
+    selectIsUnacquiredDevice,
+} from '@suite-common/wallet-core';
+import { useAlert } from '@suite-native/alerts';
+import { useTranslate } from '@suite-native/intl';
+
+type NavigationProps = StackToStackCompositeNavigationProps<
+    ConnectDeviceStackParamList,
+    ConnectDeviceStackRoutes.ConnectDeviceCrossroads,
+    RootStackParamList
+>;
+
+export const useDeviceConnect = () => {
```

Comment:
What was the purpose of changing the name from `useConnectDevice` to `useDeviceConnect`? It is not very clear from the name what the hook does.

Reply:
I was doing some refactoring and changed it because I had 2 hooks at the time and this name seemed better. Then I deleted the other one but kept the name as I think it's pretty much same when there is just one hook.